examples/js/OrientedImageHelper.js

```diff
@@ -0,0 +1,108 @@
+/* global itowns, document */
+function transformTexturedPlane(plane, distance, sizeX, sizeY, focale, ppaX, ppaY) {
+    // compute picture size
+    var Xreel = (sizeX * distance) / focale;
+    var Yreel = (sizeY * distance) / focale;
+
+    // compute ppa
+    var ppaXReel = ((ppaX - (sizeX / 2)) * distance) / focale;
+    var ppaYReel = (-(ppaY - (sizeY / 2)) * distance) / focale;
+
+    // set position and scale
+    plane.scale.set(Xreel, Yreel, 1);
+    plane.position.set(ppaXReel, ppaYReel, distance);
+}
+
+// set object position to the coordinate
+// set object ENH orientation: X to the east, Y (green) to the north, Z (blue) look to the sky.
+function placeObjectFromCoordinate(object, coord) {
+    // set object position to the coordinate
+    coord.xyz(object.position);
+    // set ENH orientation, looking at the sky (Z axis), so Y axis look to the north..
+    object.lookAt(coord.geodesicNormal.clone().add(object.position));
+}
+
+function createTexturedPlane(textureUrl) {
+    var texture;
+    var geometry;
+    var material;
+
+    texture = new itowns.THREE.TextureLoader().load(textureUrl);
+    texture.flipY = false;
+    geometry = new itowns.THREE.PlaneGeometry(1, 1, 32);
+    material = new itowns.THREE.MeshBasicMaterial({
+        map: texture,
+        color: 0xffffff,
+        side: itowns.THREE.BackSide,
+        transparent: true,
+        opacity: 1,
+    });
+    return new itowns.THREE.Mesh(geometry, material);
+}
+
+function setupCamera(camera, coord, objectToLookAt) {
+    camera.position.copy(coord.xyz());
+    camera.up.copy(coord.geodesicNormal);
+    camera.lookAt(objectToLookAt.getWorldPosition());
+}
+
+// eslint-disable-next-line no-unused-vars
+function initPicture(view, image, coord, up, lookAt, rotMatrix, distance, sizeX, sizeY, focale) {
+    var coordView;
+    var localSpace;
+    var orientedImage;
+    var quaternion;
+    var plane;
+
+    coordView = new itowns.Coordinates(view.referenceCrs, 0, 0, 0);
+    coord.as(view.referenceCrs, coordView);
+
+    // create 'local space', with the origin placed on 'coord',
+    // with Y axis to the north, X axis to the east and Z axis as the geodesic normal.
+    localSpace = new itowns.THREE.Object3D();
+    view.scene.add(localSpace);
+    placeObjectFromCoordinate(localSpace, coordView);
+
+    // add second object : 'oriented image'
+    orientedImage = new itowns.THREE.Object3D();
+    // place the 'oriented image' in the 'local space'
+    localSpace.add(orientedImage);
+
+    // setup initial convention orientation.
+    orientedImage.up.copy(up);
+    orientedImage.lookAt(lookAt);
```

Comment:
Why are you using the camera properties (`lookAt` comes from `pictureInfos.camera.lookAt`) to orient the plane?

The plane and the camera should look to each other, that might explain why you need to use `side: itowns.THREE.BackSide`...

Reply:
pictureInfos.camera.up and lookAt are used to define an axises where rotation will be applied.
Maybe I should change the name to 'pictureInfos.camera.orientationLookAt and orientationUp' ?